### DISCUSSION TITLE
Plugin dependencies not in documentation

### DISCUSSION THREAD

**Discussion by SunderApps** [PrismJS/prism#3487] [General]
Hey, so I noticed that the documentation doesn't mention that some plugins have dependencies.

- The [Download Button plugin](https://prismjs.com/plugins/download-button/) doesn't mention that it requires the [Toolbar plugin](https://prismjs.com/plugins/toolbar/) like the [Copy Button plugin](https://prismjs.com/plugins/copy-to-clipboard/) does.
- The [Previewers plugin](https://prismjs.com/plugins/previewers/) requires 'prismjs/components/prism-css-extras'.
- The [Prism Line Numbers plugin](https://prismjs.com/plugins/line-numbers/) requires the [Normalize Whitespace plugin](https://prismjs.com/plugins/line-numbers/) to display correctly.

**Comment by mAAdhaTTah** [comment]
We don't want consumers to have to think about dependencies & resolving them manually but want them instead to use one of our tools to resolve those for them. We don't list dependencies for languages either.

I don't think the 3rd bullet is a required dependency though.
